<launch>
    <!--Turtlesim Node-->
    <arg name="turtle1" default="turtle1"/>
    <arg name="turtle2" default="turtle2"/>
    <node pkg="turtlesim" type="turtlesim_node" name="turtlesim"/>
    <node pkg="roslearning" type="_03Kill_Turtle1.py" name="kill_turlte1"/>
    <node pkg="turtlesim" type="turtle_teleop_key" name="teleop" output="screen"/>

    <node name="turtle1_tf_broadcaster" pkg="roslearning" type="_01TF_Broadcaster.py" respawn="false" output="screen">
        <param name="turtle" type="string" value="$(arg turtle1)"/>
    </node>

    <node name="turtle2_tf_broadcaster" pkg="roslearning" type="_01TF_Broadcaster.py" respawn="false" output="screen">
        <param name="turtle" type="string" value="$(arg turtle2)"/>
    </node>

    <node pkg="roslearning" type="_02TF_Listener.py" name="listener">
        <param name="turtle2" type="string" value="$(arg turtle2)"/>
        <param name="turtle1" type="string" value="$(arg turtle1)"/>
    </node>

</launch>